init/sentry.js

```diff
@@ -0,0 +1,13 @@
+import * as Sentry from "@sentry/browser";
+Sentry.init({
+   dsn: "https://c16443e39a66eae141954dfd23890812@o144358.ingest.sentry.io/4505832903147520",
```

Comment:
Should the DSN be read from a config/env file?

Reply:
> DSNs are safe to keep public because they only allow submission of new events and related event data; they do not allow read access to any information. (https://docs.sentry.io/product/sentry-basics/dsn-explainer/)

Reply:
We could move to a more complex setup later, but for now, it should be fine.